src/components/separator/separator.scss

```diff
@@ -12,5 +12,5 @@ $separator: null !default;
 %ecl-separator {
   border-width: 0;
   border-top: 1px solid map.get($separator, 'color');
-  margin: var(--s-xs) 0;
+  margin: 0;
```

Comment:
was it a request to remove this spacing?
display is a little different now, I wonder if that would not cause issues
![image](https://github.com/user-attachments/assets/97c3c0f5-c909-4503-ad0d-e2cefee7cce0)
![image](https://github.com/user-attachments/assets/82f71baf-1450-4b46-b3aa-dea46f360dc0)

Reply:
yes, it was requested by the designers, i've asked ewpp what impact this would have been before applying the change, they told me that they set a bigger margin on their own on the concerned elements, so it would not been an issue. I have been warning also the designers about this but they insisted.